docs/workstation_release_management.rst

```diff
@@ -28,48 +28,66 @@ On release day (usually at least a couple weeks after releasing your first rc pa
 Step 1: Create a release candidate (rc) tag
 -------------------------------------------
 
-1. Create a release branch in the repo of the component you want to release.
-2. Push a changelog commit.
-3. Push an rc tag in the format ``<major>.<minor>.<patch>~rcN`` on your new commit. We will be building from this tag in the next step.
+1. Create a release branch (eg. ``release/1.2.3``) in the repo of the component you want to release.
+2. Update the component version to the intended release version (not the RC!) using the ``update_version.sh`` script.
```

Comment:
I don't think we should be doing this, the version should match the package version, aka 1.2.3-rcX. The only reason we can get away with this right now is because the d/changelog is in a separate repo, but we're going to fix that (c.f. https://github.com/freedomofpress/securedrop-builder/issues/308).

Bigger picture, I think this is coming from a desire eliminate the need to change the component repo, but using an arguably incorrect version number IMO goes too far and pushes us into confusing rather than helpful. We should prioritize being comfortable with diffing packages so we can verify the changes being made are what we want instead of avoiding making changes. (And we're going to lose this ability when `debian/` moves into the component repo anyways, so optimizing for it doesn't make sense)